src/main.rs

```diff
@@ -56,6 +69,14 @@ struct Opts {
     hard_linebreaks: bool,
 }
 
+// fn comma_separated(input: &str) -> Result<Vec<String>> {
+//     Ok(if input.is_empty() {
+//         Vec::new()
+//     } else {
+//         input.split(",").map(|s| s.trim().to_string()).collect()
+//     })
+// }
```

Comment:
I'm guessing you meant to implement this to support `--only-tags "foo, bar, baz"`?

Reply:
yes but for the love of god I couldn't make it work, shifting my brain to get rust to compile is quite different from haskell (which I am most familiar with) or scala/typescript (which I am using at work).